src/app/_components/common/H6.tsx

```diff
@@ -0,0 +1,17 @@
+import React, { type CSSProperties } from "react";
+import { cn } from "~/app/lib/utils";
+
+interface Props {
+  children: React.ReactNode;
+  className?: string;
+  style?: CSSProperties;
```

Comment:
We should use className for component styling and avoid multiple props like className, style and so on